kDriveCore/Data/UploadQueue/Operation/UploadOperation+UploadChunk.swift

```diff
@@ -87,18 +87,16 @@ extension UploadOperation {
                                     chunksToGenerateCount: chunksToGenerateCount)
     }
 
-    func storeChunk(_ buffer: Data, number: Int64, uploadFileId: String, sessionToken: String,
+    /// Store a chunk at the root of NSTemporaryDirectory with a stable name
+    func storeChunk(_ buffer: Data, number: Int64,
+                    uploadFileId: String,
+                    sessionToken: String,
                     hash: String) throws -> URL {
-        // Create subfolders if needed
-        let tempChunkFolder = buildFolderPath(fileId: uploadFileId, sessionToken: sessionToken)
-        Log.uploadOperation("using chunk folder:'\(tempChunkFolder)' ufid:\(uploadFileId)")
-        if !fileManager.fileExists(atPath: tempChunkFolder.path, isDirectory: nil) {
-            try fileManager.createDirectory(at: tempChunkFolder, withIntermediateDirectories: true, attributes: nil)
-        }
+        // Store chunks at the root of NSTemporaryDirectory
+        let tempRoot = URL(fileURLWithPath: NSTemporaryDirectory())
```

Comment:
Why don't you use `FileManager.default.temporaryDirectory` ?

Reply:
Yeah good point. Make sense to get an URL directly there.